Kudu.Services/Deployment/OneDeployHelper.cs

```diff
@@ -134,6 +136,27 @@ public static bool EnsureValidStaticPath(ref string path, out string error, out
             return true;
         }
 
+        public static string CustomWarName(string path)
+        {
+            string designatedRootAbsolutePath = $"/home/{WwwrootDirectoryRelativePath}/";
+
+            // Only matches files names (i.e app.war, anyname.war)
+            // Will not match relative paths (webapps/app.war, webapps/ROOT)
+            string warPattern = @"^\w+\.war$";
```

Comment:
TIL the `\w` class in regular expressions does not include the minus sign (`-`), which is common in file names (eg. `spring-petclinic-2.3.1.BUILD-SNAPSHOT.war`). Please consider using `[\w-]+` instead of `\w+` in the regex ([MDN link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes#types))

Reply:
Hmm interesting, TIL. Thanks!